### DISCUSSION TITLE
Timers not working after power loss because time resets

### DISCUSSION THREAD

**Discussion by masterofobzene** [arendst/Tasmota#21336] [Q&A] [answered]
Everything remains configured, except time. The device goes to default 1970 when power is lost.

PS: devices are not exposed to internet.

**Comment by Jason2866** [comment]
Tasmota needs access to an NTP server. If you don't provide one in your local network it acts like you see.

**Reply by sfromis** [reply, marked as answer]
Tasmota can also get real time if you add a supported RTC or GPS chip.

**Comment by HofeBY** [comment]
Install DS3231 on one ESP32 device & offer NTP Service to other devices.

**Reply by sfromis** [reply]
Tasmota should also support being a NTP server with a RTC chip on ESP8266.
